content/user/ui/settings/Integrations.md

```diff
@@ -1,12 +1,15 @@
 <!--
 title: "Integrations"
-description: "Adding Integrations to TeamServer"
+description: "Adding Integrations to Contrast"
 tags: "integrations hipchat generic slack webhook teamserver bugtracker bugzilla jira serena"
 -->
 
+
+To authorize and connect tools with Contrast to streamline your workflows, log in to your Contrast account as an **Organization Administrator**. From the user dropdown menu, select **Organization Settings** and then the **Integrations** tab in the left navigation.
```

Comment:
We should make sure we refer to the User menu in a consistent way. (I've always capitalized it.) If there is a reason not to, I'll defer to you but just noting it as something to pay attention to. Thanks!